_data/projects/SSNFS.yml

```diff
@@ -0,0 +1,11 @@
+name: SSNFS
+desc: A Simple Secure Network File System built using FUSE with the help of Qt.
+site: https://github.com/MDTech-us-MAN/SSNFS
+tags:
+- C++
+- qt
+- fuse
+- Filesystem
+upforgrabs:
+  name: help wanted
+  link: https://github.com/MDTech-us-MAN/SSNFS/issues
```

Comment:
This should be a filtered list of issues, not the whole list. For example, `https://github.com/MDTech-us-MAN/SSNFS/labels/help%20wanted`.